src/tutorials/Data-Structures/05.md

```diff
@@ -31,17 +31,14 @@ A merkle tree, or simple "hash tree," is a tree data structure in which every no
                             +----------+  +----------+
 ```
 
-In a merkle tree, nodes point to other nodes by content address (hash). Remember,
+In a merkle tree, nodes point to other nodes by their content addresses (hashes). (Remember,
 when we run data through a cryptographic hash, we get back a "hash" or "content address"
-that we can think of as a link, so a merkle tree is a collection of linked nodes.
+that we can think of as a link, so a merkle tree is a collection of linked nodes.)
 
-As previously discussed, all content addresses are unique. In the graph above, `node E` contains a reference to the hash for `node F` and `node G`.
-This means that the content address (hash) of `node E` is unique to a node containing those addresses.
+As previously discussed, all content addresses are unique. In the graph above, `node E` contains a reference to the hash for `node F` and `node G`. This means that the content address (hash) of `node E` is unique to a node containing those addresses.
 
 Getting lost? Let's imagine this as a set of file folders. If we run folder E through our hashing algorithm
-while it contains subfolders F and G, the content-derived hash we get back is unique to its current content.
-If we remove folder G, it's like Grace removing that whisker from her kitten photo. The folder formerly known as E doesn't
-have the same contents anymore, so it gets a new hash.
+while it contains subfolders F and G, the content-derived hash we get back will include references to those two folders. If we remove folder G, it's like Grace removing that whisker from her kitten photo. Folder E doesn't have the same contents anymore, so it gets a new hash.
```

Comment:
I simplified “the folder formerly known as E” to “folder E.” Is this fudgy-ness okay, since we’re dealing in a metaphor here? The core idea is intact, i think: “change the contents of the folder, and the folder changes, too.” The current phrase implies that the name of each node is shorthand for its hash, but that’s not explicit. And might not be necessary info(?)